crates/floresta-electrum/Cargo.toml

```diff
@@ -11,28 +11,30 @@ description = """
 repository = "https://github.com/vinteumorg/Floresta"
 license = "MIT"
 readme.workspace = true # Floresta/README.md
-
 keywords = ["bitcoin", "utreexo", "node", "blockchain", "rust"]
 categories = ["cryptography::cryptocurrencies"]
 
 [dependencies]
-floresta-common = { path = "../floresta-common" }
-floresta-chain = { path = "../floresta-chain" }
-floresta-watch-only = { path = "../floresta-watch-only" }
-floresta-compact-filters = { path = "../floresta-compact-filters" }
-floresta-wire = { path = "../floresta-wire" }
-
-rustreexo = "0.4"
-tokio = { version = "1.0", features = ["full"] }
-tokio-rustls = "0.26"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-bitcoin = { version = "0.32", features = ["serde", "std", "bitcoinconsensus"] }
+bitcoin = { workspace = true, features = ["serde", "std", "bitcoinconsensus"] }
```

Comment:
Do `electrum` need `bitcoinconsensus`? Could you try to remove this feature and see if it compiles?